scripts/util/remake_utils.py

```diff
@@ -74,6 +75,34 @@ def write_shortcuts(service, chartname, id, age, new_folders, alias_map):
         
         # Make the shortcut
         util.make_shortcut(service, partfile_name, partfile.get('id'), new_folders['sec'][age][part])
+    
+    # Create a shortcut for each part in the Audio folder
+    res = util.get_drive_files(service, audio_id)
+    if not res or len(res) == 0:
+        print(f'WARNING: No audio files found for chart "{chartname}"')
+        return
+    
+    for audiofile in res:
+        # Ensure audio part exists in the system
+        check_stop_script()
+        audiofile_name = audiofile.get('name')
+        _, part, _ = util.parse_file(audiofile_name, alias_map)
+        if part == None:
+            print(f'WARNING: Audio file "{partfile_name}" has no matching part folder')
+            continue
+
+        # Make the shortcut
+        util.make_shortcut(service, audiofile_name, audiofile.get('id'), new_folders['aud'][age][part])
+
+# Changes the name of all the old folders to 
+def rename_old_folders(service, sep_ids):
+    for abbr, full in [('old', 'Old'), ('curr', 'Current'), ('fut', 'Future')]:
+        for loc in ['sec', 'aud', 'sib']:
+            try:
+                service.files().update(fileId=sep_ids[f'{abbr}_{loc}'],
```

Comment:
`service.files().update(fileId=sep_ids[f'{loc}_{abbr}'], body={'name': f'{full} Chartz (To be Deleted)'}).execute()`